syntax = "proto3";

package NGenerativeBoltalka.Proto;

message TEmbeddingResponse {
    repeated float Embedding = 1;
}
